conda/recipes/libcugraph/conda_build_config.yaml

```diff
@@ -62,3 +62,6 @@ cuda11_cuda_profiler_api_host_version:
 
 cuda11_cuda_profiler_api_run_version:
   - ">=11.4.240,<12"
+
+libucxx_version:
+  - "0.43.*"
```

Comment:
This needs a corresponding entry in `update-version.sh`, so this version will be updated when new release branches are cut. Here's how RAFT does that: https://github.com/rapidsai/raft/blob/68d412a119f918dc003ecd0c430b6bde7499bc96/ci/release/update-version.sh#L77

Alternatively, since this is only being used in 1 place, you could forgo the `conda_build_config.yaml` entry and just directly put the version in `conda/recipes/libcugraph/meta.yaml`, like this:

```yaml
- libucxx =0.43.*
```

I recommend just doing that because it's simpler, but either would be fine.

Reply:
Went ahead and pushed the `update-version.sh` bits to Chuck's PR